### PR TITLE
Single select - Fix margin, border and radius of ghost dropdown

### PR DESCRIPTION
**Pull request by harellevy**

### CODE REVIEW

**Comment by harellevy** [comment]
image

**Comment by harellevy** [comment]
image

File: projects/ui-framework/src/lib/lists/single-select/single-select.component.html [hunk 1]
```diff
@@ -48,8 +48,8 @@
 
 <ng-template #ghostTmp>
   <div class="ghost">
-    <strong class="mrg-r-8">{{label}}</strong> <span>{{displayValue}}</span>
-    <span class="mrg-l-8 select-chevron b-icon-medium"
+    <strong class="label">{{label}}</strong> <span class="value">{{displayValue}}</span>
+    <span class="select-chevron b-icon-medium"
```
Comment (galloween) [inline projects/ui-framework/src/lib/lists/single-select/single-select.component.html:52]: i saw in design sometimes they use it without label, so maybe you should add ngIf
and if you also add ngIf on the 'value' you dont need the css, you can keep the mrg-l-8

File: projects/ui-framework/src/lib/lists/single-select/single-select.component.ts [hunk 2]
```diff
@@ -111,6 +111,15 @@ export class SingleSelectComponent extends BaseSelectPanelElement {
     return option?.value;
   }
 
+  ngOnChanges(changes) {
+    if (this.ghost) {
+      this.panelClassList.push('panel-ghost');
+    } else {
+      this.panelClassList = [...this.panelClassList.filter(className => className !== 'panel-ghost')];
```
Comment (galloween) [inline projects/ui-framework/src/lib/lists/single-select/single-select.component.ts:118]: you dont need the [... 
:)

Comment (galloween) [reply projects/ui-framework/src/lib/lists/single-select/single-select.component.ts:118]:
```
if (changes.ghost) {
  this.panelClassList = this.panelClassList.filter(className => className !== 'panel-ghost');
  this.ghost &&  this.panelClassList.push('panel-ghost');    
}
```

File: projects/ui-framework/src/lib/lists/single-select/single-select.component.html [hunk 3]
```diff
@@ -48,9 +48,14 @@
 
 <ng-template #ghostTmp>
   <div class="ghost">
```
Comment (galloween) [inline projects/ui-framework/src/lib/lists/single-select/single-select.component.html:50]: this div you can remove

**Review by galloween — APPROVED** [review]

**Pull request merged**